docs/src/main/asciidoc/getting-started-guide.adoc

```diff
@@ -262,13 +270,20 @@ public class GreetingResource {
     @GET
     @Produces(MediaType.TEXT_PLAIN)
     public String hello() {
-        return "hello\n";
+        return "hello";
     }
 }
 ----
 
 If you stopped the application, restart the application with `./mvnw compile quarkus:dev`.
-Then check that http://localhost:8080/hello/greeting/quarkus returns `hello quarkus`.
+Then check that the endpoint returns `hello quarkus` as expected:
+
+```
+$ curl http://localhost:8080/hello/greeting/quarkus
```

Comment:
I would use `-w "\n"` in this example.

Reply:
I kept it out to avoid having to repeat it everywhere - but I guess I could add it and then remove the [tip] section about curl ?

Reply:
I don't think it's an issue. I would keep the tip and add it to the 2 `curl` instances.

If it was a problem big enough for someone to create a PR to fix it, I think it's worth having a clean behavior.

Reply:
okey - fixed in latest commit.